syntax = "proto2";

package novact.fc.msg;

message SensorBaro {
  required int64 timestamp_us = 1; // Timestamp in microseconds
  required float pressure_pa = 2; // Pressure in Pascals
  required float temperature_c = 3; // Temperature in Celsius
  required float altitude_m = 4; // Estimated altitude in meters
}